src/os/FileJournal.cc

```diff
@@ -1719,6 +1721,9 @@ bool FileJournal::read_entry(
     &ss);
   if (result == SUCCESS) {
     if (next_seq > seq) {
+      if (next_seq - 1 > journaled_seq) {
+        journaled_seq = next_seq -1;
+      }
       return false;
     } else {
       read_pos = next_pos;
```

Comment:
I think we should usd next_seq instead of seq. 
Such as next_seq is 100, and we call do_read_entry which set seq is 90. Now jouranl_replay finish. We should use 99 instead of 90.